cluster/clean.sh

```diff
@@ -15,6 +15,7 @@ $kubectl delete --ignore-not-found -f $MANIFESTS_DIR/operator.yaml
 $kubectl delete --ignore-not-found -f $MANIFESTS_DIR/nmstate.io_nodenetworkconfigurationenactments_crd.yaml
 $kubectl delete --ignore-not-found -f $MANIFESTS_DIR/nmstate.io_nodenetworkconfigurationpolicies_crd.yaml
 $kubectl delete --ignore-not-found -f $MANIFESTS_DIR/nmstate.io_nodenetworkstates_crd.yaml
+$kubectl delete --ignore-not-found -f $MANIFESTS_DIR/nmstate.io_nmstates_crd.yaml
```

Comment:
We are skipping the nmstate controller, let's not create/delete the CRDs for now.